functions/go/apply-setters/README.md

```diff
@@ -0,0 +1,94 @@
+# apply-setters
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Apply setter values on resources fields. May set either the complete or partial field value.
+
+<!--mdtogo-->
+
+Setters provide a solution for template-free setting of field values. They are a
+safer alternative to other substitution techniques which do not have the context
+of the structured data. Setters may be invoked to modify the configuration
```

Comment:
we seem to be using 'resource' and 'configuration' interchangeably. It's pervasive in all our docs and is confusing. As part of auditing command references, we should try to make things consistent.

Reply:
Agree. We should use `resource` word to refer to resource files.